linux/buildScript.sh

```diff
@@ -11,12 +11,12 @@ set -u
 
 srctgz="https://www.census.gov/ts/x13as/unix/x13ashtmlsrc_V1.1_B19.tar.gz"
 file=$(basename ${srctgz})
-
 td=$(tempfile -p x13)
+rm -r $td
```

Comment:
And in the newer script we have `td=$(mktemp -d -p /tmp x13dirXXXXXX)` which pretty much guarantees a new directory.